Sources/NostrSDK/Events/NonParameterizedReplaceableEvent.swift

```diff
@@ -16,4 +16,20 @@ public extension NonParameterizedReplaceableEvent {
 
         return try? EventCoordinates(kind: kind, pubkey: publicKey, relayURL: relayURL)
     }
+
+    func shareableEventCoordinates(relayURLStrings: [String]? = nil, excludeAuthor: Bool = false, excludeKind: Bool = false) throws -> String {
```

Comment:
Generally it is less cognitive load for `Bool`s to be positively stated: `includeAuthor`, `includeKind`. That way you don't have to deal with double-negative like you would with `!excludeAuthor`.

Reply:
That's good feedback. My brain is hardwired to always have the default parameters be false, which is why I went on the `exclude` route. But doing the inverse makes more sense in this case.